examples/envs/lung/lung_pid.ipynb

```diff
@@ -0,0 +1,182 @@
+{
```

Comment:
Could we split into three files with relevant names? (e.g., `examples/envs/lung/balloon_lung.ipynb`; leading underscore removed because this is a "public" example not a "private" internal code file)